Libraries/LibWeb/CSS/Properties.json

```diff
@@ -1123,7 +1123,8 @@
     "initial": "auto",
     "valid-types": [
       "length [-∞,∞]",
-      "percentage [-∞,∞]"
+      "percentage [-∞,∞]",
+      "anchor"
```

Comment:
The spec says that `anchor()` is invalid outside of the "inset properties", which is not only top/right/bottom/left but also `inset-block(-start/end)`, `inset-inline(-start/end)` and `inset`.

Reply:
My understanding is that these logical aliases end up getting mapped to a one of the top/bottom/left/right physical properties, so we should be able to use `anchor()` with them, as-is:

Tested with the following, which shows that the property does get set:
```html
<!DOCTYPE html>
<body></body>
<script>
    const element = document.createElement("div");
    const property =  "inset-inline-start";
    const value = "anchor(inside)";
    element.style[property] = value;
    console.log(`Specified value for ${property}: "${element.style[property]}"`);
</script>
```

Reply:
Oh that's sweet! I could've used that with `anchor-size()` as well, instead of manually checking the property context 🤔